services/inc/enumflags.h

```diff
@@ -264,6 +267,12 @@ inline bool particle::EnumFlags<T, typename std::enable_if_t<std::is_enum<T>::va
     return (value_ <= flags.value_);
 }
 
+template<typename T>
+inline particle::EnumFlags<T>& particle::EnumFlags<T, typename std::enable_if_t<std::is_enum<T>::value>>::operator=(EnumFlags<T> flags) {
+    std::swap(*this, flags);
+    return *this;
+}
```

Comment:
Nit: A common idiom for swapping objects in C++ is
```
using std::swap; // Enable ADL
swap(obj1, obj2);
```

Reply:
Fixed